raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -356,9 +356,9 @@ def test_get_lock_amount_after_fees_imbalanced_channel(
     integers(min_value=0, max_value=100),
     integers(min_value=0, max_value=10_000),
     integers(min_value=0, max_value=50_000),
-    integers(min_value=1, max_value=90_000),
-    integers(min_value=1, max_value=100_000),
-    integers(min_value=1, max_value=100_000),
+    integers(min_value=1, max_value=90_000_000_000_000_000),
```

Comment:
The increase in the amounts is to showcase the precision errors right? So without the changes of this PR the test fails due to lack of precision?

Reply:
Yes!